gulpfile.js

```diff
@@ -40,7 +40,7 @@ gulp.task('serve', ['build-dev'], function() {
   gulp.watch(paths.styles.src + '/**/*', ['styles']);
   gulp.watch(paths.assets.src + '/**/*', ['copy-assets']);
   gulp.watch(paths.script.src + '/**/*.js', ['lint', 'scripts']);
-  gulp.watch(paths.base + '/index.html', ['copy-html']);
+  gulp.watch(paths.base + '/*.{html,js,json}', ['copy-html']);
```

Comment:
Do we need the manifest to be hot reloaded?

The SW script should be moved to `./js` and be picked up by the `scripts`/`dist` tasks.